### ISSUE TITLE
Breaking change between 0.2.4 and 1.1.3

### ISSUE THREAD

**Issue by ZebraFlesh** [davidcalhoun/jstoxml#31]
When consuming the 0.2.4 release through webpack 3 using babel-loader, a default module export is generated: 
![screen shot 2017-10-28 at 1 19 52 pm](https://user-images.githubusercontent.com/10542617/32137325-47de5a4c-bbe3-11e7-9fc7-3a29d63e9082.png)

Consuming the 1.1.3 release through the same version of webpack results in no default module export: 
![screen shot 2017-10-28 at 1 23 27 pm](https://user-images.githubusercontent.com/10542617/32137336-6354466a-bbe3-11e7-8254-6717b32df577.png)

This forces an imports change from:

```
import jstoxml from 'jstoxml';
```
to
```
import { toXML } from 'jstoxml';
```

Looking at the different method of computing module exports in the 1.x release line, it's not immediately clear to me why this is happening. (Maybe I'm just not up on my module semantics.)

Submitting this in case this behavior change wasn't intentional, but mostly as a documentation trail for others.

**Comment by davidcalhoun** [comment]
Thanks for the heads up!  That's weird, it's not entirely clear to me why that's happening either.  In 1.x the library is now wrapped in a UMD wrapper (about the first 13 lines at the top), which Webpack may be consuming differently.

**Comment by davidcalhoun** [comment]
Thanks again for the heads up.  This is now fixed in `1.3.0`
